src/Common/src/Interop/Windows/Gdi32/Interop.CreateCompatibleDC.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Gdi32
+    {
+        [DllImport(Libraries.Gdi32, ExactSpelling = true)]
```

Comment:
Did you intentionally remove `SetLastError` from a bunch of these `PInvokes`?

Reply:
Yeah where we don’t check any errors we don’t need this interop overhead